README.md

```diff
@@ -329,6 +335,7 @@ question instructions.
     *a. this is the **first**
     *b. this is the **second**
     c. this is the **third**
+    ?d. choosing this will not affect to the granted points
```

Comment:
`choosing this will not affect to the granted points` should be `choosing this does not affect the granted points`

Reply:
There should also be an example of a partial points question. When you add a new question, update the max points in the beginning of the questionnaire too.